src/NServiceBus.Core/Recoverability/Faults/ErrorQueueSettings.cs

```diff
@@ -23,19 +29,31 @@ public static string ErrorQueueAddress(this ReadOnlySettings settings)
         {
             string errorQueue;
 
-            if (settings.TryGet("errorQueue", out errorQueue))
+            if (TryGetErrorQueueAddress(settings, out errorQueue))
             {
-                Logger.Debug("Error queue retrieved from code configuration via 'EndpointConfiguration.SendFailedMessagesTo().");
                 return errorQueue;
             }
 
+            return DefaultErrorQueueName;
+        }
+
+        internal static bool TryGetErrorQueueAddress(this ReadOnlySettings settings, out string errorQueue)
+        {
+            if (settings.HasExplicitValue(SettingsKey))
+            {
+                Logger.Debug("Error queue retrieved from code configuration via 'EndpointConfiguration.SendFailedMessagesTo().");
+                errorQueue = settings.Get<string>(SettingsKey);
+                return true;
+            }
+
             var section = settings.GetConfigSection<MessageForwardingInCaseOfFaultConfig>();
             if (section != null)
             {
                 if (!string.IsNullOrWhiteSpace(section.ErrorQueue))
                 {
                     Logger.Debug("Error queue retrieved from <MessageForwardingInCaseOfFaultConfig> element in config file.");
-                    return section.ErrorQueue;
+                    errorQueue = section.ErrorQueue;
+                    return true;
```

Comment:
As discussed in the meeting, I'm referring to the fact that one of the remediation options is to remove the invalid configuration that caused the exception, and rely on the default error queue value instead.